content/direct-deposit-management/release-notes/2025-06-18

```diff
@@ -0,0 +1,4 @@
+We fixed an issue so that Veterans with a military address can successfully update their direct deposit account information. 
```

Comment:
@lglover-va, it might be helpful to explain what a military address is. I don't know if the developers using this API would know what that means - it might be helpful to define it?